src/app/components/PageTitleConfirmation.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Element, EventListener } from '@openstax/types/lib.dom';
+
+export default function announcePageTitle(title: string) {
+  const el = document?.getElementById('page-title-confirmation');
+
+  el?.dispatchEvent(
+    new CustomEvent('page-title', {
+      detail: title,
+    })
+  );
+}
```

Comment:
I feel like this function could be a callback that calls setDocTitle directly instead of dispatching the event and using the listener

Reply:
You might need to create a `usePageTitleConfirmation` pattern for that though.

Reply:
The only way I could see doing that was to use a Context, and I found this lighter and less coupled. There might be a more clever way to feed setDocTitle to the announce function so it could be called.